<ui version="4.0" >
 <class>screenshotMakerDialog</class>
 <widget class="QDialog" name="screenshotMakerDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Take a screenshot</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>File</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" colspan="2" >
       <widget class="QCheckBox" name="autoFileExtension" >
        <property name="text" >
         <string>Automatically select filename extension</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QToolButton" name="fileButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QComboBox" name="fileBox" >
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="insertPolicy" >
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close|QDialogButtonBox::NoButton|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>View</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QComboBox" name="viewChooser" >
        <item>
         <property name="text" >
          <string>Top</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Front</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Side</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>3D</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QStackedWidget" name="previewStack" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="widget" />
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QGroupBox" name="qualityBox" >
     <property name="title" >
      <string>Quality</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QSlider" name="qualitySlider" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>90</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QSpinBox" name="qualitySpinbox" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>90</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Image Format</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="fileType" >
        <property name="currentIndex" >
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="byFileExtension" >
        <property name="text" >
         <string>By file extension</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>3</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Resolution</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="4" column="3" >
       <widget class="QToolButton" name="removeButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Remove from Resolutions&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
        <property name="icon" >
         <iconset resource="../actions.qrc" >:/add-delete/media/edit_remove.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="2" >
       <widget class="QToolButton" name="addButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Add to resolutions&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
        <property name="icon" >
         <iconset resource="../actions.qrc" >:/add-delete/media/add.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Resolution:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3" >
       <widget class="QComboBox" name="resolutionCombo" >
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex" >
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3" >
       <widget class="QSpinBox" name="heightBox" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="accelerated" >
         <bool>true</bool>
        </property>
        <property name="maximum" >
         <number>4094</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3" >
       <widget class="QSpinBox" name="widthBox" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="accelerated" >
         <bool>true</bool>
        </property>
        <property name="maximum" >
         <number>4049</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>1280</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4" >
       <widget class="QCheckBox" name="costumResolution" >
        <property name="text" >
         <string>Costum Resolution</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2" >
       <widget class="QCheckBox" name="aspectRatio" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Keep aspect ratio</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>viewChooser</tabstop>
  <tabstop>fileType</tabstop>
  <tabstop>byFileExtension</tabstop>
  <tabstop>qualitySlider</tabstop>
  <tabstop>qualitySpinbox</tabstop>
  <tabstop>resolutionCombo</tabstop>
  <tabstop>costumResolution</tabstop>
  <tabstop>widthBox</tabstop>
  <tabstop>heightBox</tabstop>
  <tabstop>aspectRatio</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>fileBox</tabstop>
  <tabstop>fileButton</tabstop>
  <tabstop>autoFileExtension</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../actions.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>qualitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>qualitySpinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>706</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel" >
     <x>706</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qualitySpinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>qualitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>706</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel" >
     <x>706</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewChooser</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>previewStack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>76</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel" >
     <x>78</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previewStack</sender>
   <signal>currentChanged(int)</signal>
   <receiver>viewChooser</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>181</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>costumResolution</sender>
   <signal>clicked(bool)</signal>
   <receiver>widthBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>706</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel" >
     <x>706</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>costumResolution</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>706</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel" >
     <x>567</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>costumResolution</sender>
   <signal>clicked(bool)</signal>
   <receiver>heightBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>706</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel" >
     <x>706</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>costumResolution</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>706</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel" >
     <x>567</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>costumResolution</sender>
   <signal>clicked(bool)</signal>
   <receiver>aspectRatio</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>706</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel" >
     <x>642</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>costumResolution</sender>
   <signal>clicked(bool)</signal>
   <receiver>resolutionCombo</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>706</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel" >
     <x>706</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>byFileExtension</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileType</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>651</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel" >
     <x>656</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>screenshotMakerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>364</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
